modules/transformer.py

```diff
@@ -96,17 +96,18 @@ def create_html_bracket(bracket_map):
     finals = []
     playoff_map = {}
 
-    if west == [[]] or east == [[]]:
-        playoff_map = {
-        "wc1" : [],
-        "wc2" : [],
-        "wcf" : [],
-        "ec1" : [],
-        "ec2" : [],
-        "ecf" : [],
-        "f" : []
+    if west == [[]] or east == [[]]: # if returned scraped data is empty, need to return an empty map with correct fields
```

Comment:
You can just change this to `if not west[0] or not east[0]:` no need to create a new list for conditional.